tests/console/sle11_consoletest_setup.pm

```diff
@@ -44,7 +43,7 @@ sub run() {
 
     become_root;
     script_run "chmod 444 /usr/sbin/packagekitd";    # packagekitd will be not executable
-    script_run "exit";
+    type_string "exit\n";
 
     save_screenshot;
```

Comment:
I am thinking this might be dangerous as a screenshot is saved but we can not be sure if the "exit" has already been executed at this time.